doc/lua_api.txt

```diff
@@ -4858,6 +4858,8 @@ Utilities
           dynamic_add_media_table = true,
           -- allows get_sky to return a table instead of separate values (5.6.0)
           get_sky_as_table = true,
+          -- "zstd" method for compress/decompress (master)
```

Comment:
"(master)"? should be (5.6.0) (next release = current master)

Reply:
1032f7b now changes this to 5.6.0